utils/profile_tool/sub.py

```diff
@@ -18,10 +18,11 @@ def command_sub(args):
 
     exclusive_rules = len(subtracted_profile.get_rule_selectors())
     exclusive_vars = len(subtracted_profile.get_variable_selectors())
-    if exclusive_rules > 0:
-        print("{} rules were left after subtraction.".format(exclusive_rules))
-    if exclusive_vars > 0:
-        print("{} variables were left after subtraction.".format(exclusive_vars))
+    if exclusive_rules > 0 or exclusive_vars > 0:
+        print(
```

Comment:
This print could be moved to be inside the conditional in line 27. The criteria is the same.